### DISCUSSION TITLE
"Outflow" boundary in premixed sphere flames

### DISCUSSION THREAD

**Discussion by Alex-jian522** [AMReX-Combustion/PeleLMeX#338] [Q&A]
Hello, I'm using PeleLMeX for a 2D premixed sphere flame. And I set the boudary conditions as follow (all four boundary conditions to be "outflow"):
#----------------------DOMAIN DEFINITION------------------------
geometry.is_periodic = 0 0               # For each dir, 0: non-perio, 1: periodic
geometry.coord_sys   = 0                  # 0 => cart, 1 => RZ
geometry.prob_lo     = 0.0 0.0 0.0        # x_lo y_lo (z_lo)
geometry.prob_hi     = 0.01 0.01 0.016        # x_hi y_hi (z_hi)

#>>>>>>>>>>>>>  BC FLAGS <<<<<<<<<<<<<<<<
peleLM.lo_bc = Outflow Outflow
peleLM.hi_bc = Outflow Outflow

But from the velocity (and ave_pressure) contours, I found that the velocity at the four corners of the two-dimensional computational domain is almost 0, which is quite strange.
![image](https://github.com/AMReX-Combustion/PeleLMeX/assets/72745362/fed58554-a570-4e3a-9494-8702d31c9fa9)

Is there something wrong with my boundary condition settings? Are there more suitable boundary conditions in PeleLMeX? Thank you very much for any reply.

**Comment by baperry2** [comment]
Relevant discussion from PeleC for reference: https://github.com/AMReX-Combustion/PeleC/discussions/753#discussioncomment-8388086
